modules/miner/blockmanager_test.go

```diff
@@ -0,0 +1,105 @@
+package miner
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// reconstructBlock reconstructs a block given the miner and the header
+func reconstructBlock(m *Miner, header types.BlockHeader) (block *types.Block, err error) {
+	block, exists := m.blockMem[header]
+	if !exists {
+		err = errors.New("Header is either invalid or too old")
+		return
```

Comment:
Coming back to this, you'll find examples of Sia using named returns all over the codebase. This is because we stopped using named return values within the past few weeks, much of the codebase hasn't been ported to the new strategy.